client/go.mod

```diff
@@ -1,5 +1,22 @@
 module github.com/kubernetes-csi/volume-data-source-validator/client
 
-go 1.16
+go 1.19
 
-require k8s.io/apimachinery v0.22.0
+require k8s.io/apimachinery v0.26.0
```

Comment:
@bswartz Is it okay to update this without running other scripts for the client?